src/convertToObject.js

```diff
@@ -3,16 +3,25 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
+  const sourceArray = sourceString.split(';');
+
+  sourceArray.forEach(element => {
+    const [key, value] = element.split(':').map(style => style.trim());
```

Comment:
map is not needed here because you have only  two elements key and value. And you are creating a new array via `map`, which is unnecessary